src/arvtool.c

```diff
@@ -394,6 +399,164 @@ arv_tool_control (int argc, char **argv, ArvDevice *device)
         }
 }
 
+static void
+arv_tool_network (int argc, char **argv, ArvDevice *device)
+{
+        if (!ARV_IS_GV_DEVICE (device))
+                printf ("This is not a GV device\n");
+        else if (argv[2] != NULL) {
```

Comment:
I would prefer if there was a loop here, like in the 'control' command.

And the ip subcommand changed to only take an ip address.

Reply:
I agree with that. After some more thoughts I think this is a better solution.